ui/app/AppLayouts/Wallet/AssetsTab.qml

```diff
@@ -1,8 +1,10 @@
 import QtQuick 2.13
 import "../../../imports"
 import "../../../shared"
+import "../../../shared/polyfill.number.toLocaleString.js" as NumberPolyFill
 
 Item {
+    property var appSettings
```

Comment:
It's not necessary to pass the `appSettings` down the children. It's globally accessible. I used to pass the `appSettings` myself, but that was before I knew we could just access the parent's property directly. It's easier.